Makefile

```diff
@@ -122,10 +114,7 @@ _apply_controller_cfg:
 	$(TOOL) apply -f ./deploy -n $(NAMESPACE)
 	mv ./deploy/role_binding.yaml.bak ./deploy/role_binding.yaml
 ifeq ($(TOOL),oc)
-ifeq ($(WEBHOOK_ENABLED),true)
-	$(TOOL) apply -f ./deploy/os/che-workspace-controller-cert-gen-deployment.yaml -n $(NAMESPACE)
```

Comment:
The *other* cert gen is for running on k8s, since it doesn't support the service annotation. I thought I got everything related to this cert gen but I'll double-check. We'll eventually need to decide how we want to support k8s, but for now I think leaving the existing tools in place is a good idea.

Reply:
Looking at your diff -- it looks like I lost some changes in a rebase. I was *sure* I removed those files :)

Reply:
Cherry-picked your changes to the branch. Thanks for prepping a working commit :)